.travis.yml

```diff
@@ -59,6 +63,7 @@ matrix:
       env: TARGET=go
 
 before_install:
+  - ./.travis/before-install-mvn3.3.9-trusty.sh
```

Comment:
@ericvergnaud why you ignore the first line on travis file:
```dist: trusty```
If we only have one travis file, this setting would be global for every target.